src/everest/config/export_config.py

```diff
@@ -36,6 +36,8 @@ class ExportConfig(BaseModel, extra="forbid"):  # type: ignore
 
     @field_validator("csv_output_filepath", mode="before")
     @classmethod
-    def validate_output_file_writable(cls, csv_output_filepath):  # pylint:disable=E0213
+    def validate_output_file_writable(
+        cls, csv_output_filepath: str | None
+    ) -> str | None:
         check_writable_filepath(csv_output_filepath)
```

Comment:
Why not just return None and not force to pass None to `check_writable_filepath`?

Reply:
The subsequent functions raise a Exception on None. Returning None would change behavior